dev_tools/ownership_utils.py

```diff
@@ -16,8 +16,8 @@ def __init__(self, pattern: str, owners: Tuple[str, ...], line_number: int) -> N
 
 
 class GithubOwnerShip:
-    def __init__(self, repo_dir: Path) -> None:
-        self._ownerships = parse_ownership(repo_dir / ".github" / "CODEOWNERS")
+    def __init__(self, repo_dir: Path, codeowners_file: Path) -> None:
```

Comment:
Should we make the second parameter defaulting to `".github" / "CODEOWNERS"`? Then we don't have to modify all tests

Reply:
It would need to be defaulted to `repo_dir / ".github" / "codeowners"`, but `repo_dir` is another argument. I checked that it won't work.

Reply:
on second thought, why not. Let me try ;p